parsers/CA_AB.py

```diff
@@ -79,9 +79,11 @@ def fetch_price(
     prices = PriceList(logger)
     for row in csv.reader(response.text.split("\r\n\r\n")[2].splitlines()[1:]):
         if row[1] != "-":
+            date, hour = row[0].split()
             prices.append(
                 zoneKey=zone_key,
-                datetime=arrow.get(row[0], "MM/DD/YYYY HH", tzinfo=TIMEZONE).datetime,
+                datetime=datetime.strptime(f"{date} {int(hour) - 1}", "%m/%d/%Y %H")
+                + timedelta(hours=1),
```

Comment:
```suggestion
                datetime=datetime.strptime(f"{date} {int(hour) - 1}", "%m/%d/%Y %H").replace(tzinfo=TIMEZONE),
                + timedelta(hours=1),
```

We need to attach a the timezone as well.

Also are you sure that the value of hour 24 are for 00 the next day and not simply hour 23 but 1 indexed? Otherwise we don't need the timedelta.